control/control.CASP.xml

```diff
@@ -441,13 +441,6 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
                 <enable_next>no</enable_next>
             </defaults>
             <modules config:type="list">
-                <!-- As soon as possible -->
-                <module>
-                    <label>Installer Update</label>
-                    <name>update_installer</name>
-                    <enable_back>no</enable_back>
```

Comment:
I guess the disabled going back (`enable_back`) should be moved to the new first item.